pkg/storage/mysql/mysql.go

```diff
@@ -111,7 +111,7 @@ func NewWithDB(db *sql.DB, cfg *sqlcommon.Config) (*MySQL, error) {
 	}
 
 	stbl := sq.StatementBuilder.RunWith(db)
-	dbInfo := sqlcommon.NewDBInfo(db, stbl, sq.Expr("NOW()"))
+	dbInfo := sqlcommon.NewDBInfo(db, stbl, sq.Expr("NOW()"), HandleSQLError)
```

Comment:
What do you think if we rename this function `HandleMySQLError`, then for postgres `HandlePostgresError`?

This is based on this article about code searchability: https://morizbuesing.com/blog/greppability-code-metric/

Reply:
I'm not sure about other IDEs but at least in vscode it's able to distinguish between them and show only the correct one when searching for references. Even if you are grepping for strings any references outside the module would be `mysql.HandleSQLError` or `postgre.HandleSQLError` so I tend to think that keeping the same name across them makes more sense as they are implementing the same callback.

The other argument for keeping them the same is that the 2 implementations are very close to identical, If you compare mysql.go to postgres.go there are actually far more diffs due to "noise" from things like `m` vs `p` when defining methods than from actual differences in logic. I'd argue that we'll have an easier time maintaining these 2 backends if we reduce that (and would be happy to put together a PR) rather than adding to it by having a diff in every pl;ace that we call `HandleSQLError`.